src/main/java/org/jitsi/jigasi/transcription/AbstractTranscriptPublisher.java

```diff
@@ -194,15 +195,45 @@ protected void sendMessage(ChatRoom chatRoom, T message)
         try
         {
             chatRoom.sendMessage(chatRoomMessage);
-            if (logger.isDebugEnabled())
-                logger.debug("Sending message: \"" + messageString + "\"");
+            if (logger.isTraceEnabled())
+                logger.trace("Sending message: \"" + messageString + "\"");
         }
         catch (OperationFailedException e)
         {
             logger.warn("Failed to send message " + messageString, e);
         }
     }
 
+    /**
+     * Send a json-message to the muc room
+     *
+     * @param chatRoom the chatroom to send the message to
+     * @param jsonMessage the json message to send
+     */
+    protected void sendJsonMessage(ChatRoom chatRoom, T jsonMessage)
+    {
+        if (chatRoom == null)
```

Comment:
maybe you can check already here if chatroom is of the right type and return when it's not